README.rst

```diff
@@ -191,6 +192,8 @@ how to go about it.
 Changelog
 ---------
 
+- v1.0.0b
+    - Remove Python 2 support, see `Issue 64 <https://github.com/niccokunzmann/python-recurring-ical-events/issues/64>`_.
```

Comment:
Looks like you're also removing 3.5 and 3.6 support, so that should also be documented.

Reply:
Thanks, yes that should be done!